flow/templates/drexel-picotte.sh

```diff
@@ -0,0 +1,28 @@
+{% extends "slurm.sh" %}
+{% set partition = partition|default('standard', true) %}
+{% block tasks %}
+    {% set threshold = 0 if force else 0.9 %}
+    {% set cpu_tasks = operations|calc_tasks('np', parallel, force) %}
+    {% set gpu_tasks = operations|calc_tasks('ngpu', parallel, force) %}
+    {% if gpu_tasks and 'gpu' not in partition and not force %}
+        {% raise "Requesting GPUs requires a gpu partition!" %}
+    {% endif %}
+    {% set nn_cpu = cpu_tasks|calc_num_nodes(48) if 'gpu' not in partition else cpu_tasks|calc_num_nodes(48) %}
+    {% set nn_gpu = gpu_tasks|calc_num_nodes(4) if 'gpu' in partition else 0 %}
+    {% set nn = nn|default((nn_cpu, nn_gpu)|max, true) %}
+    {% if partition == 'gpu' %}
+#SBATCH --nodes={{ nn|default(1, true) }}
+#SBATCH --ntasks-per-node={{ (gpu_tasks, cpu_tasks)|max }}
```

Comment:
Does this correctly handle multi node GPU submissions?

Reply:
@b-butler Perhaps — are you thinking that we are missing “divide tasks by the computed number of nodes and round up”? If you see something else, please clarify.

I believe this logic is copied from other templates so we should check carefully.

Reply:
Yes, we do this below for the `dft` partition. We do this in GreatLake's template, but I think we also error there. However, we don't deal with multi node submissions there since we don't have enough resource assess for that. An example of the other is the Expanse template.

Reply:
This will be handled separately in #566.